app/models/task.rb

```diff
@@ -2,4 +2,21 @@
 
 class Task < ActiveRecord::Base
```

Comment:
Should this class do any validations to ensure that no instances are created with `completed: true`?